### ISSUE TITLE
Issue with magic-ext/auth in react native?

### ISSUE THREAD

**Issue by thatportugueseguy** [magiclabs/magic-js#583]
Hello, 

I'm developing an app with react native and wanted to integrate with Magic, but i'm hitting an issue with `magic-ext/auth` where i get an error that the imported file can't be found. Most specifically: `...dist/react-native/index.native.js`.

I'm fairly new to developing in react native, so there might be something that i missed here, but it seems that this line is pointing react native in the wrong direction, as there isn't any dist build directory with the name `react-native`. I have looked at the expo packages and they don't have this `react-native` prop.

The issue seems to be with the following line in `package.json`: 

https://github.com/magiclabs/magic-js/blob/276bad5d201c4c37a86e7d914f8373ff84edb81f/packages/%40magic-ext/auth/package.json#L20

At the moment, the fix i have for it is pointing the `react-native` prop to the `index.js` file, or simply deleting the line completely.

Am i doing something wrong when installing this module? AFAIU, I don't think so. 

Also, it seems that there is something similar on the `webauthn` extension too. There is a `index.native.ts` file, but the reference doesn't seem to be pointing to the right file, and it seems that there isn't a `dist/react-native` folder too

https://github.com/magiclabs/magic-js/blob/276bad5d201c4c37a86e7d914f8373ff84edb81f/packages/%40magic-ext/webauthn/package.json#L20

Please let me know if these lines are something leftover from a refactor or something of the like, and if you'd like me to submit a PR pointing those lines to the right files, or changing them in some other way.

Thank you

**Comment by nicomanzato** [comment]
I'm having the same issue, is there a workaround or a fix coming up for this issue?

**Comment by ayv8er** [comment]
Did you guys react to the deprecation notice when you call any of the auth methods, and use the auth extension?

**Comment by thatportugueseguy** [comment]
Yes, this is using the auth extension, but has nothing to do with the deprecation notice. It's a different error.

**Comment by ayv8er** [comment]
Is this reproducible when using Magic SDK without the auth extension?

**Comment by sherzod-bakhodirov** [comment]
Hey, thanks for reporting this.

I’m going to close the issue for now since it’s quite old. If you’re still running into this problem, feel free to open a new issue with current details and we’ll take another look.
